### DISCUSSION TITLE
银联卡支付不了，账户马上到期，急！

### DISCUSSION THREAD

**Discussion by henrylyubj** [getlantern/lantern-forum-cn#595] [Q&A]
![Uploading WeChatcc3116a468430f2f2bbdcb892199c2c8.jpg…]()

**Comment by MaxMKMax** [comment]
@henrylyubj 您好. 请您提供我您的账号邮箱和重新发刚才的截图给我. 我帮您查看. 谢谢!

**Comment by henrylyubj** [comment]
***@***.***
这两天感觉被严重限速了，例如这张图只有45KB，上传一分钟...打开邮件也很慢，我还有2天就到期了

Mike K ***@***.***> 于2024年6月17日周一 23:58写道：

**Comment by henrylyubj** [comment]
刚才又尝试使用银联，还是不行

**Comment by henrylyubj** [comment]
能不能给我的账号延期一些时间，等贵公司找到妥善的支付方式，我支付后，再扣除延期的时间？

HENRY LYU ***@***.***> 于2024年6月18日周二 21:30写道：

**Reply by MaxMKMax** [reply]
@henrylyubj 我帮您问了. 看能帮您紧急处理. 但我不知道您的账号邮箱. 您空 m i k e @ get lantern . or g 邮箱写给我.

**Comment by MaxMKMax** [comment]
@henrylyubj 您好. 请您发邮件给我吧. 我这里完全看不到您的信息. 发账号邮箱和支付单据给我. 谢谢!
mike@getlantern.org

m i k e @ get lantern . or g

**Reply by henrylyubj** [reply]
我昨天已经给您发送邮件了

**Reply by MaxMKMax** [reply]
@henrylyubj 您好. 我帮您申请免费 3 个月了. 很抱歉影响了您的体验感. 您先用着, 过 1 个星期再是看是否可以充值. 谢谢您支持了我们 8 年. 很开心又像您这么尊贵的用户.

**Reply by henrylyubj** [reply]
好像不行啊，没有收到3个月的延期

**Reply by henrylyubj** [reply]
您好，我的账号好像没有收到3个月的延期，期间也一直尝试用各种手段支付，都未能成功

**Reply by MaxMKMax** [reply]
@henrylyubj 您好. 好的, 我帮您查问看看.

**Comment by henrylyubj** [comment]
感谢贵公司的慷慨。说实话，我这两天一直在尝试使用银联支付，至少几十次，就在刚才还在尝试。但到最后一步还是不行，我感觉我实在找不到什么好办法。我觉得贵公司应该开发一个新的安全的支付方式。

Mike K ***@***.***> 于2024年6月19日周三 23:37写道：

**Reply by MaxMKMax** [reply]
@henrylyubj 对的. 第三方支付系统有些问题导致支付宝,银联无法使用. 我们在解决这问题了. 谢谢您的耐心等待.

**Comment by henrylyubj** [comment]
尊敬的Mike ***@***.***
，谢谢。现在登录界面上显示是非专业用户。

Mike K ***@***.***> 于2024年6月19日周三 23:37写道：

**Comment by MaxMKMax** [comment]
@henrylyubj 您好. 已经添加好 3 个月专业版给您了.

**Comment by henrylyubj** [comment]
好的，好像可以了，感谢您和您的同事

Mike K ***@***.***> 于2024年6月25日周二 01:17写道：

**Reply by MaxMKMax** [reply]
@henrylyubj 
亲, 不客气呢. 谢谢您多年来对我们的支持. 请您过 1 到 2 周后再试试看支付. 有什么我会再更新的. 

如果您不介意的话, 可以到应用点给我们点 5 一个星好评和留言. 谢谢您. 感激不尽!

**Comment by henrylyubj** [comment]
亲爱的Mike先生：
***@***.***
贵公司赠送我的三个月免费期限马上又要到期，期间我又尝试使用银联卡支付，还是不成功。我在用户论坛上看到用户可以通过“单独生成的支付链接”付费，能不能给我这个邮箱也发一个链接，我想赶紧续费，最好能使用支付宝。

顺祝商祺
Mike K ***@***.***> 于2024年6月26日周三 01:04写道：

**Comment by henrylyubj** [comment]
亲爱的Mike先生：
我的账号是 henrytownbj@gmail.com
贵公司赠送我的三个月免费期限马上又要到期，期间我又尝试使用银联卡支付，还是不成功。我在用户论坛上看到用户可以通过“单独生成的支付链接”付费，能不能给我这个邮箱也发一个链接，我想赶紧续费，最好能使用支付宝。
顺祝商祺！
